AudioKit/Platforms/Common/CsoundObj.m

```diff
@@ -142,11 +172,15 @@ - (void)record:(NSString *)csdFilePath toURL:(NSURL *)outputURL
 
 - (void)record:(NSString *)csdFilePath toFile:(NSString *)outputFile
 {
+#ifdef TRAVIS_CI
+    [self performSelectorOnMainThread:@selector(runCsoundToDisk:) withObject:@[csdFilePath, outputFile] waitUntilDone:YES];
```

Comment:
No idea, and its strange, it will work some of the time, maybe 10%?  And the failures are a variety of unexplained things.  With performSelector it works every time.